geodirectory-functions/map-functions/map_functions.php

```diff
@@ -55,12 +90,34 @@ function send_marker_jason_to_js()
                     $canvas_jason = '[{"totalcount":"0"}]';
 
                 $map_canvas_jason_args = array($canvas . '_jason' => $canvas_jason);
+
+                /**
+                 * Filter for map canvas json args
+                 *
+                 * You can use this filter to modify map canvas json args.
+                 *
+                 * @since 1.0.0
+                 * @package GeoDirectory
+                 * @param string $canvas map canvas array key.
+                 * @param array $map_canvas_jason_args map canvas args.
+                 */
                 $map_canvas_jason_args = apply_filters('geodir_map_canvas_jason_' . $canvas, $map_canvas_jason_args);
 
                 wp_localize_script('geodir-map-widget', $canvas . '_jason_args', $map_canvas_jason_args);
             } else {
                 $canvas_jason = '[{"totalcount":"0"}]';
                 $map_canvas_jason_args = array($canvas . '_jason' => $canvas_jason);
+
+                /**
+                 * Filter for map canvas json args

```

Comment:
it would be good to mentions in the summary or description that this filter is used in the send_marker_jason_to_js() function.